.flowconfig

```diff
@@ -1,6 +1,7 @@
 [ignore]
 ; We fork some components by platform
 .*/*[.]android.js
+.*/*[.]macos.js
```

Comment:
Don't want to litter the review with the same comment, but we should probably tag any mac-specific diffs from fb's master so merge resolution can be more intentional